Web/starter_project_vanguard/components/story/Partners.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+const Partners = () => {
+  return (
+    <div className="max-w-screen-lg mt-20">
+      <h1 className="font-DMSans text-5xl font-semibold mb-16 text-center">
+        Current Interview Partners
+      </h1>
+      <div className="flex flex-col sm:flex-row  gap-1">
+        <img
```

Comment:
make this in a loop

you can have a list like:
```
const partners = [
    {companyName: "Google", imageUrl: "Google.png"},
    {companyName: "Palantir", imageUrl: "Palantir.png"},
    ...
]
```

then loop over it to populate the img using it

Reply:
[nit] also might be worth checking out if there is an image component implemented by tailwind (might have better responsiveness and stuff built into it)